developer-documentation/wallet-sdk.md

```diff
@@ -1,3 +1,118 @@
 # Wallet SDK
 
 The [Wallet SDK](https://github.com/docknetwork/wallet-sdk-demo) enables you to build a Verifiable Credentials wallet inside your app and allows your users to receive, store, and manage their DOCK tokens too. This was built for native applications with added support for Polkadot-JS.&#x20;
+
+For using the wallet-sdk, all you need to do is wrap your app in a `WalletSDKProvider` and start building your wallet.
+
+## Installation
+```js
+yarn add @docknetwork/wallet-sdk-core
```

Comment:
given we've taken these down from npm, are these instructions still valid?

Reply:
now it should be valid again